packages/grafana-sql/src/components/configuration/ConnectionLimits.tsx

```diff
@@ -1,5 +1,5 @@
 import { DataSourceSettings } from '@grafana/data';
-import { ConfigSubSection, Stack } from '@grafana/experimental';
+import { ConfigSubSection, Stack } from '@grafana/plugin-ui';
```

Comment:
does the stack component exist in plugin-ui? iirc from my pr i had to import it from grafana/ui instead